src/Sulu/Component/Rest/ListBuilder/Filter/FilterTypeRegistry.php

```diff
@@ -18,7 +18,7 @@ class FilterTypeRegistry
      */
     private $filterTypes;
 
-    public function __construct(iterable $filterTypes)
+    public function __construct(\Traversable $filterTypes)
```

Comment:
`iterator_to_array` would fail when a non \Traversable is given. Only PHP 8.0 supports "unpacking" of array and traversable via `this->filterTypes = [...$filterTypes];`